### ISSUE TITLE
the screenshot is not showed

### ISSUE THREAD

**Issue by pduyet** [fescobar/allure-docker-service#223]
I run test and use API to send reports to allure docker service. but the screenshot cannot show
![image](https://github.com/fescobar/allure-docker-service/assets/39377321/2baf96ee-816a-4231-adb8-f0e84e5fbff9)
on my local, the report generate by allure serve still appears the screenshot

**Comment by victor-wang-anz** [comment]
hey @pduyet , we have a same problem here.
according to the swagger spec, each file in the allure-results folder must be converted to a base64 for string.
I am wondering if this rule also applies to the screenshot.png file?

**Comment by pduyet** [comment]
> hey @pduyet , we have a same problem here. according to the swagger spec, each file in the allure-results folder must be converted to a base64 for string. I am wondering if this rule also applies to the screenshot.png file?
> 
> @fescobar is it something possible or we cannot upload png images through the send result api?

yes, I see that rule. we cannot convert the attachment file png in allure-results to base64. it makes the screenshot cannot show on the report. I'm studying about this issue

**Comment by fescobar** [comment]
Hi @victor-wang-anz @pduyet 
What is the file that you want to send? Please, can you send me the example?
I don't have any issues with any image. Did you try using the scripts provided here? https://github.com/fescobar/allure-docker-service?tab=readme-ov-file#send-results-through-api
